src/qhttp/AjaxEndpoint.cc

```diff
@@ -24,23 +24,31 @@
 #include "qhttp/AjaxEndpoint.h"
 
 // Local headers
+#include "lsst/log/Log.h"
+#include "qhttp/LogHelpers.h"
 #include "qhttp/Request.h"
 #include "qhttp/Response.h"
 #include "qhttp/Server.h"
 
+namespace {
+    LOG_LOGGER _log = LOG_GET("lsst.qserv.qhttp");
+}
+
 namespace lsst {
 namespace qserv {
 namespace qhttp {
 
 
-AjaxEndpoint::AjaxEndpoint()
+AjaxEndpoint::AjaxEndpoint(std::shared_ptr<Server> server)
+:
+    _server(server)
 {
 }
 
 
 AjaxEndpoint::Ptr AjaxEndpoint::add(Server& server, std::string const& path)
 {
-    auto aep = std::shared_ptr<AjaxEndpoint>(new AjaxEndpoint);
+    auto aep = std::shared_ptr<AjaxEndpoint>(new AjaxEndpoint(std::shared_ptr<Server>(&server)));
```

Comment:
Although this is not strictly required, it's a good practice to add `const` to variables that are not going to be changed:
```
auto const aep = ...
```
Reasons: self-defense (prevents accidental modifications of the variable) and code documentation (tells a reader that the variable is read-only).

Reply:
Sensible; will do